modules/ml/src/main/java/org/apache/ignite/ml/math/impls/matrix/AbstractMatrix.java

```diff
@@ -996,4 +996,8 @@ protected int getMaxAmountOfColumns(double[][] data) {
 
         return maxAmountOfCols;
     }
+
+    @Override public String toString() {
```

Comment:
Maybe better use showAscii operator and print out first k rows and k columns of matrix (if < 100)